packages/relay/tests/lib/eth/eth_call.spec.ts

```diff
@@ -440,6 +440,71 @@ describe('@ethCall Eth Call spec', async function () {
         'Error invoking RPC: Invalid contractCallResponse from consensus-node: undefined',
       );
     });
+
+    it('should handle SDKClientError in callConsensusNode', async function () {
```

Comment:
this is to be removed in PR https://github.com/hiero-ledger/hiero-json-rpc-relay/issues/3918.

Reply:
removed